tests/checkers/unittest_similar.py

```diff
@@ -178,6 +181,19 @@ def test_ignore_nothing():
     )
 
 
+def test_lines_without_meaningful_content_do_not_trigger_similarity():
+    output = StringIO()
+    with redirect_stdout(output), pytest.raises(SystemExit) as ex:
+        similar.Run([SIMILAR3, SIMILAR4])
+    assert ex.value.code == 0
+    assert (
+        output.getvalue().strip()
+        == """
+TOTAL lines=18 duplicates=0 percent=0.00
```

Comment:
Hello @Pierre-Sassoulas ,
Thank you for taking a look at the PR. I ran your examples through the standard similarity checker (before my modification) and they did not trigger a similarity there---I believe at least 5 lines of identical code is needed (the number of identical lines has to be greater than the minimum of 4, not just greater than or equal to).

I do want to also make sure that the goal is clear and agreed on. According to the conversation in the issue thread with @AWhetter, we would not want to trigger a warning on 5 identical lines if at least one of the lines is just symbols---because lines with just symbols should not count towards the 5 line limit.

You were completely correct in that my unit test was weak though.  I added an example of a case where a grouping of code still should trigger a warning and found that I needed to change `re.match` to `re.search` .  I hope the re-submitted test looks better to you.  Let me know if you think there still should be additional changes or impromevents.
Thanks,
Eli

Reply:
Goob job on changing the test, it's very nice now. I agree that the line with nothing or only a symbol should be subtracted from the line count (ie: 8 lines with 3 "only a symbol line" = 5 lines).